magma/backend/mlir/mlir_to_verilog.py

```diff
@@ -33,6 +33,6 @@ def mlir_to_verilog(
         )
         pm.run(module.operation)
     if opts.split_verilog:
-        circt.export_split_verilog(module, "")
+        circt.export_split_verilog(module, ".")
```

Comment:
Should add a test to cover this case

Reply:
done. also added an option to pass in the directory since that might be useful